src/test/java/com/training/api/tests/ga/GuestAccountTests.java

```diff
@@ -194,4 +195,59 @@ public void testNegativeLoginAuthenticationWrongEmail(){
         softAssertions.assertThat(responseElement.getErrors().get(0).getDeveloperMessage().length()).as(ERRORS_MESSAGE)
                 .isGreaterThan(0);
     }
+
+    @Test
+    public void testCreateGuestAccount(){
+        String id = RandomStringUtils.randomAlphanumeric(10);
```

Comment:
There is still a very minute possibility of getting a duplicate id.
A full proof mechanism would be generate id using a timestamp which would be always be unique.